test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1525,6 +1525,16 @@ def forward(self, x):
         x = torch.randn(2, 1, 4)
         self.run_test(Squeeze(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(13)
+    def test_squeeze_dynamic_axes(self):
```

Comment:
I think this should be more clear if it were named `_dynamic_dim` not `dynamic_axes`. The arg to squeeze is named `dim`, and `dynamic_axes` usually refers to the argument to `torch.onnx.export`

Please also edit the PR title and commit messages accordingly.